client/src/components/GlobalStyledComponents/Container.jsx

```diff
@@ -1,10 +1,14 @@
-import React from 'react';
 import styled from 'styled-components';
 
 export const Container = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-evenly;
+
   width: 100%;
-  padding-right: 15px;
-  padding-left: 15px;
+  padding-right: 1rem;
+  padding-left: 1rem;
```

Comment:
lets move those rems to the spaces variables

couple of things about `rems` in general

1. they are useful. basically when user zooms the screen. thats it. as far as I know.
2. they are annoying. since 1 rem is the default font size of the browser, unless you state your default in html rule, and that is usually a number hard to divide or increment.
3. used for font size, and sometimes padding + margin.
4. Im never sure if to work with them or not